torchgeo/models/dofa.py

```diff
@@ -387,12 +387,7 @@ def forward(self, x: Tensor, wavelengths: list[float]) -> Tensor:
 
 # https://github.com/zhu-xlab/DOFA/blob/master/normalize_dataset.py
 # Normalization is sensor-dependent and is therefore left out
-_dofa_transforms = K.AugmentationSequential(K.CenterCrop((224, 224)), data_keys=None)
-
-# https://github.com/pytorch/vision/pull/6883
-# https://github.com/pytorch/vision/pull/7107
-# Can be removed once torchvision>=0.15 is required
-Weights.__deepcopy__ = lambda *args, **kwargs: args[0]
+_dofa_transforms = nn.Sequential(T.CenterCrop((224, 224)))
```

Comment:
wrapping in `nn.Sequential` instead of `T.Compose` allows the transforms to be exportable